OSDOCS#16913: CQA for next set of CLI assemblies (#107352)
openshift/openshift-docs: modules/about-cli-profiles-switch.adoc

:_mod-docs-content-type: CONCEPT
[id="about-switches-between-cli-profiles_{context}"]
= About switching between CLI profiles

[role="_abstract"]
You can use contexts to easily switch between users across multiple {product-title} clusters when using the {oc-first}. You specify a nickname to make managing CLI configurations easier by providing short-hand references to contexts, user credentials, and cluster details.

> After a user logs in with the `oc` CLI for the first time, {product-title} creates a `~/.kube/config` file if one does not already exist. As more authentication and connection details are provided to the CLI, either automatically during an `oc login` operation or by manually configuring CLI profiles, the updated information is stored in the configuration file:

Reviewer: ```suggestion
After a user logs in with the `oc` CLI for the first time, {product-title} creates a `~/.kube/config` file if one did not already exist. Over time, more authentication and connection details are stored in the configuration file:
```
do we need those how details here? i feel like probably not?

[NOTE]
====

> If you have access to administrator credentials but are no longer logged in as the default system user `system:admin`, you can log back in as this user at any time as long as the credentials are still present in your CLI configuration file. The following command logs in and switches to the default project:

Reviewer: ```suggestion
If you have access to administrator credentials but are not logged in as the default system user `system:admin`, you can log back in as this user as long as the credentials are present in your CLI configuration file.

The following command logs in and switches to the default project:
```
